src/cascadia/WindowsTerminal/WindowsTerminal.manifest

```diff
@@ -18,7 +18,7 @@
         <!-- You apparently are expected to include all the maxversiontested
         entries separately. They are treated like an array. This one turns on Segoe
         UI Variable, GH #12452 -->
-        <maxversiontested Id="10.0.22000.0"/>
+        <maxversiontested Id="10.0.22621.0"/>

```

Comment:
You must _add_ another `maxversiontested`, not replace the existing one :smile: (see comment above)